gui/src/app/CompileContext/CompileContextProvider.tsx

```diff
@@ -0,0 +1,86 @@
+import { ProjectContext } from "@SpCore/ProjectContextProvider";
+import compileStanProgram from "@SpStanc/compileStanProgram";
```

Comment:
thoughts on moving the `compileStanProgram.ts` file into this folder?

Reply:
Done